### PR TITLE
LPS-175545 | Test Fix and Refactor | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Test FiX
This fix is corresponding to these tickets:
- [LPS-175544](https://issues.liferay.com/browse/LPS-175544)
- [LPS-175545](https://issues.liferay.com/browse/LPS-175545)

# Refactor
This refactor is corresponding to these testcases:
- SegmentationCreateSegment.testcase
- SegmentationCreateSegment.testcase

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase [hunk 1]
```diff
@@ -853,11 +843,8 @@ definition {
 		}
 	}
 
-	@description = "LPS-167023 Given a user with Segments permissions
-	And with View Control Panel Menu permission
-	When the user clicks on Applications Menu > Control Panel > Segment
-	Then the user can access to the Segments page"
-	@priority = "5"
+	@description = "LPS-167023: Validate if the user with View Control Panel Menu permission can access to the Segments page"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase:846]: For the description, just put the original description but in one line

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase (hunk 1 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase]: done!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:forward

**Pull request closed without merging**